content/oci/oci.go

```diff
@@ -270,6 +269,10 @@ func (s *Store) SaveIndex() error {
 	s.indexLock.Lock()
 	defer s.indexLock.Unlock()
 
+	return s.saveIndex()
+}
+
+func (s *Store) saveIndex() error {
```

Comment:
It is better to keep as it was. Although it may cause slight perf loss in `DeletableStore`, I would prefer better encapsulation as `DeletableStore` is not designed to be perf optimized.

Reply:
Changed `SaveIndex()` back to its original version.